.github/workflows/test.yml

```
@@ -125,6 +131,11 @@ jobs:
         run: docker run --network=host -e "PGUSER=postgres" gcr.io/ggl-cas-storage/cas-cif-schema:${{ github.sha }} sqitch deploy
       - name: start app
         run: docker run -d --network=host -e "PGUSER=postgres" -p 3004:3004 gcr.io/ggl-cas-storage/cas-cif-app:${{ github.sha }}
+      - name: start minio
+        run: docker run -d --network=host -p 9000:9000 minio/minio:RELEASE.2021-04-22T15-44-28Z server /data
+      - name: start storage api
+        run: | # pragma: allowlist secret
+          docker run -d --network=host -e "STORAGE_API_KEY=mXZpQpJjGIyQsRRViScPfeOItOQJokod" -e "MINIO_ACCESS_KEY=minioadmin" -e "MINIO_SECRET_KEY=minioadmin" -e "MINIO_HOST_URL=minio:9000" -p 8000:8000 gcr.io/ggl-cas-storage/cas-cif-storage:${{ github.sha }}
```

```suggestion
          docker run -d --network=host -e "STORAGE_API_KEY=_doNotUseThisApiKeyInProduction_" -e "MINIO_ACCESS_KEY=minioadmin" -e "MINIO_SECRET_KEY=minioadmin" -e "MINIO_HOST_URL=minio:9000" -p 8000:8000 gcr.io/ggl-cas-storage/cas-cif-storage:${{ github.sha }}
```
Another 32 character string that cannot be mistaken for a being a real secret